service/storage_service.cc

```diff
@@ -3755,15 +3773,15 @@ future<> storage_service::node_ops_abort_thread() {
         while (!_node_ops_abort_queue.empty()) {
             auto uuid_opt = _node_ops_abort_queue.front();
             _node_ops_abort_queue.pop_front();
-            if (!uuid_opt) {
-                slogger.info("Stopped node_ops_abort_thread");
-                co_return;
-            }
             try {
-                co_await node_ops_abort(*uuid_opt);
+                co_await node_ops_abort(uuid_opt.value_or(utils::null_uuid()));
```

Comment:
This seems not to be the part of just "fix cross-shard abort source usage". Would you extend the motivation in the commit message, please?

Reply:
sure